app/assets/stylesheets/new_design/personnes_impliquees.scss

```diff
@@ -0,0 +1,7 @@
+.personnes-impliquees {
+  padding-bottom: 50px;
+
+  ul {
```

Comment:
ajouter un margin left pour corriger l'alignement ?
<img width="136" alt="capture d ecran 2017-12-05 a 10 11 21" src="https://user-images.githubusercontent.com/847942/33598904-b24d43d2-d9a4-11e7-8c6a-0d841122131a.png">

Reply:
Je ne suis pas contre, j'y ai brièvement pensé, mais ça permet de faire des "hanging bullets" (cf. http://www.markboulton.co.uk/journal/five-simple-steps-to-better-typography-part-2, partie "Examples of Hanging Punctuation").

Après il est possible que ce traitement typographique soit à réserver aux textes et ne soit pas adapté à de l'UI :)

Reply:
Même sur les hanging bullets, ils sont alignés sur les titres non ? :D

Reply:
Je crois que c'est la légende de l'image plus que le titre du paragraphe dans l'illustration d'exemple :)

Reply:
@mmagn done